plugins/utils/comment-box.nix

```
@@ -0,0 +1,153 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}:
+let
+  inherit (lib.nixvim) defaultNullOpts;
+in
+lib.nixvim.neovim-plugin.mkNeovimPlugin config {
+  name = "comment-box";
+  originalName = "comment-box.nvim";
+  luaName = "comment-box";
+  defaultPackage = pkgs.vimPlugins.comment-box-nvim;
+  description = ''
+    Clarify and beautify your comments and plain text files using boxes and lines.
+  '';
+
+  maintainers = [ lib.maintainers.elythh ];
+
+  settingsOptions = {
+    comment_style =
+      defaultNullOpts.mkEnum
+        [
+          "line"
+          "block"
+          "auto"
+        ]
+        "line"
+        ''
+          Select the type of comments. Default: 'line'.
```

```suggestion
          Select the type of comments.
```
The default will show up in the UI because of the "line" pluginDefault declared above. We can remove all these `Default:`

---

You can see what I'm talking about if you temporarily change the defaultPlugin to something that will build and run `serve-docs` command from the devShell.